README.md

```diff
@@ -1,12 +1,10 @@
 # AWS SDK for pandas (awswrangler)
 
-AWS Data Wrangler is now **AWS SDK for pandas (awswrangler)**.  We’re changing the name we use when we talk about the library, but everything else will stay the same.  You’ll still be able to install using `pip install awswrangler` and you won’t need to change any of your code.  As part of this change, we’ve moved the library from AWS Labs to the main AWS GitHub organisation but, thanks to the GitHub’s redirect feature, you’ll still be able to access the project by its old URLs until you update your bookmarks.  Our documentation has also moved to [aws-sdk-pandas.readthedocs.io](https://aws-sdk-pandas.readthedocs.io), but old bookmarks will redirect to the new site.
-
 *Pandas on AWS*
 
 Easy integration with Athena, Glue, Redshift, Timestream, OpenSearch, Neptune, QuickSight, Chime, CloudWatchLogs, DynamoDB, EMR, SecretManager, PostgreSQL, MySQL, SQLServer and S3 (Parquet, CSV, JSON and EXCEL).
 
-![AWS SDK for pandas](docs/source/_static/logo2.png?raw=true "AWS SDK for pandas")
+![AWS SDK for pandas](https://github.com/aws/aws-sdk-pandas/blob/main/docs/source/_static/logo2.png?raw=true "AWS SDK for pandas")
```

Comment:
Don't we want this to be a "local" path. For example, if I'm on the `release/2.x.` branch, I would want the readme to display that version of the logo.

Reply:
I was just being consistent with the rest of the [README](https://github.com/aws/aws-sdk-pandas/blob/main/README.md?plain=1#L125) but perhaps we should make the change everywhere else